django/views/templates/csrf_403.html

```diff
@@ -53,7 +79,7 @@ <h2>Help</h2>
     <li>Your browser is accepting cookies.</li>
 
     <li>The view function passes a <code>request</code> to the template’s <a
-    href="https://docs.djangoproject.com/en/dev/topics/templates/#django.template.backends.base.Template.render"><code>render</code></a>
+    href="https://docs.djangoproject.com/en/{{ docs_version }}/topics/templates/#django.template.backends.base.Template.render"><code>render</code></a>
```

Comment:
This is unrelated to my changes, except that I'm in here and noticed that it wasn't using the same version format for these doc urls.

Reply:
Can do. Would the process be to create a separate ticket in Trac as well, or is that overkill for this?

Reply:
Awesome. Here's the new PR: https://github.com/django/django/pull/18180